velox/docs/functions/spark/string.rst

```diff
@@ -141,6 +141,17 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT overlay('Spark SQL', 'tructured', 2, 4); -- "Structured SQL"
         SELECT overlay('Spark SQL', '_', -6, 3); -- "_Sql"
 
+.. spark:function:: repeat(str, n) -> varchar
+
+    Returns the string which repeats the given string value ``n`` times. 
+    ``n`` must be less or equal than 10000. ::
```

Comment:
Why do we add this limitation? I tried in Spark and found a larger value for n was allowed.

Reply:
I would be better to document the behavior when n is negative.

Reply:
Updated the doc.
Spark fuzzer test failed with exit code 137 which indicates out of memory error. I thought this is related to this parameter.
 I think 10000 times is big enough, not sure if larger vaule would be use in real cases. Or we should disable this fucntion in fuzzer test, please suggest.

Reply:
@zhli1142015, could you get the input size when OOM happens? Maybe, we should limit the value of `input_size * times`. If this value is greater than some threshold, let the execution fail with exception.

Reply:
Or let's wait for the review from @mbasmanova.

Reply:
@zhli1142015, I see. The limit is an empirical value. I'm not sure how to handle. @mbasmanova, do you have any idea? Should we just let fuzzer test ignore the out-of-memory failure instead of introducing an empirical limit?

Reply:
I wonder if we can introduce a flag like `FLAGS_repeat_times` and set it as an acceptable value when running the fuzzer test to avoid OOM.

Reply:
@zhli1142015 You can reproduce fuzzer failures using --seed 2932607463. You can then debug these using standard tools.

Overall, it would be nice to add a limit on the size of the output string.

https://github.com/facebookincubator/velox/pull/10218#discussion_r1652969475